ons_alpha/jinja2/templates/pages/bulletin_page.html

```diff
@@ -44,8 +44,10 @@ <h1 class="ons-u-fs-3xl bulletin-header__heading">
                         {% endif %}
                     </div>
                     <div class="ons-grid__col ons-col-4@m">
-                        <span class="ons-u-fs-r--b bulletin-header__releases-label">{{ _("Contact") }}:</span>
-                        <a href="mailto:{{ page.contact_details.email }}">{{ page.contact_details.name }}</a>
+                        {% if page.contact_details.email %}
```

Comment:
@helenb Should we output the name, even if there isn't an email address to link to?

Reply:
Actually if there is a contact details snippet added, it will have both name and email, as both are required. I'll update the test to be just `{% if page.contact_details %}`